components/crx_file/crx_verifier_unittest.cc

```diff
@@ -7,18 +7,32 @@
 #include "base/files/file_path.h"
 #include "base/path_service.h"
 #include "base/strings/string_number_conversions.h"
+#if defined(STARBOARD)
+#include "starboard/system.h"
+#endif
 #include "testing/gtest/include/gtest/gtest.h"
 
 namespace {
 
 base::FilePath TestFile(const std::string& file) {
+#if defined(STARBOARD)
+  std::vector<char> buf(kSbFileMaxPath);
+  SbSystemGetPath(kSbSystemPathContentDirectory, buf.data(), kSbFileMaxPath);
+  return base::FilePath(buf.data())
+      .AppendASCII("test")
+      .AppendASCII("components")
+      .AppendASCII("crx_file")
+      .AppendASCII("testdata")
+      .AppendASCII(file);
```

Comment:
Isn't the buffer used only for the value of the system path content directory (i.e., the other components are appended to the base::FilePath instance, but not the buffer)?

But anyway I think it's ok since 1) as you said, it's a unit test, and 2) the appended components are known at build time.